lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -317,6 +320,14 @@ def start(self):
                 p["geomag_params"],
             ) = read_mc_dl2_to_QTable(p["file"])
 
+            
+            if self.data_bin.scale_true_energy != 1.0:
+                # This will scale the true energy of the MC events just before filling the IRFs histograms when pyirf commands are used.
+                # Effects expected: non-diagonal energy dispersion matrix and a different spectrum.
```

Comment:
Just for the sake of reference, can you show a comparative plot of the modified IRFs and Crab Nebula spectrum? Or refer to a presentation if already shown in the collaboration.

Reply:
They will be presented for the next LST-reco-call (10/06).